### PR TITLE
Add `linrongbin16/gitlinker.nvim`

### PR DESCRIPTION
**Pull request by linrongbin16**
### Repo URL:

https://github.com/linrongbin16/gitlinker.nvim

### Checklist:

- [x] The plugin is specifically built for Neovim, or if it's a colorscheme, it supports treesitter syntax.
- [x] The lines end with a `.`. This is to conform to `awesome-list` linting and requirements.
- [x] The title of the pull request is ```Add/Update/Remove `username/repo` ``` (notice the backticks around ``` `username/repo` ```) when adding a new plugin.
- [ ] The description doesn't mention that it's a Neovim plugin, it's obvious from the rest of the document. No mentions of the word `plugin` unless it's related to something else.
- [x] The description doesn't contain emojis.
- [ ] Neovim is spelled as `Neovim` (not `nvim`, `NeoVim` or `neovim`), Vim is spelled as `Vim` (capitalized), Lua is spelled as `Lua` (capitalized), Tree-sitter is spelled as `Tree-sitter`.
- [x] Acronyms should be fully capitalized, for example `LSP`, `TS`, `YAML`, etc.

### CODE REVIEW

**Comment by linrongbin16** [comment]
hi, would you tell me, how to fix the lint pipeline error?

**Comment by tversteeg** [comment]
It seems it's a broken change upstream.

**Comment by linrongbin16** [comment]
> Did you consider PR rather than a fork? It would most likely benefit more people, @linrongbin16.
> 
> @tversteeg apologies if that's not the right place to ask that question. Feel free to delete my comment.

Actually I did, before 3 months ago..., I'm not sure the author would reply.

**Comment by tversteeg** [comment]
The changes you've made don't adhere to the checklist, I've deselected the item(s) that need to be fixed.

**Comment by tversteeg** [comment]
You still have `for Neovim` in your description.

File: README.md [hunk 1]
```diff
@@ -715,6 +715,7 @@ These colorschemes may not specialize in Tree-sitter directly but are written in
 - [AckslD/nvim-gfold.lua](https://github.com/AckslD/nvim-gfold.lua) - Plugin using [gfold](https://github.com/nickgerace/gfold) to switch repo and have statusline component.
 - [akinsho/git-conflict.nvim](https://github.com/akinsho/git-conflict.nvim) - A plugin to visualise and resolve merge conflicts.
 - [aaronhallaert/advanced-git-search.nvim](https://github.com/aaronhallaert/advanced-git-search.nvim) - Search your git history by commit content, message and author with Telescope.
+- [linrongbin16/gitlinker.nvim](https://github.com/linrongbin16/gitlinker.nvim) - A plugin to generate sharable file permalinks (with line ranges) for git host websites, fork of [ruifm's gitlinker](https://github.com/ruifm/gitlinker.nvim), refactored with pattern based rule engine, windows support and other enhancements.
```
Comment (tversteeg) [inline README.md:718]:
```suggestion
- [linrongbin16/gitlinker.nvim](https://github.com/linrongbin16/gitlinker.nvim) - Generate sharable file permalinks (with line ranges) for git host websites, fork of [ruifm's gitlinker](https://github.com/ruifm/gitlinker.nvim), refactored with pattern based rule engine, windows support and other enhancements.
```

Comment (linrongbin16) [reply README.md]: Hi, my latest commit: https://github.com/rockerBOO/awesome-neovim/pull/783/commits/d7e8622d331920f3bddcc4741e380928c5a25f17 did the same thing. thanks.

**Pull request merged**